corehq/ex-submodules/casexml/apps/case/dbaccessors/by_owner.py

```diff
@@ -14,8 +32,13 @@ def get_closed_case_ids(owner_id):
 
 def _get_case_ids(owner_id, is_closed):
```

Comment:
so does `is_closed` mean:
`True` - Show only closed
`False` - Show only open
`None` - Show all

I like the convention, but I think it could use a comment